pkg/providers/nutanix/template.go

```diff
@@ -519,3 +524,26 @@ func generateNutanixFailureDomains(eksNutanixFailureDomains []v1alpha1.NutanixDa
 	}
 	return failureDomains
 }
+
+func addKubeVipToIgnoredNodeIPsList(clusterSpec *cluster.Spec, result []string) []string {
+	kubeVipStr := clusterSpec.Cluster.Spec.ControlPlaneConfiguration.Endpoint.Host
+	if kubeVipStr != "" {
+		kubeVip, err := net.ResolveIPAddr("ip", kubeVipStr)
+		if err != nil {
+			// log error and continue
+			log.Printf("error resolving kube-vip IP address %s: %v", kubeVipStr, err)
```

Comment:
Can we perform this check in the validations and just use the value here directly?
Maybe you can add it here https://github.com/aws/eks-anywhere/blob/main/pkg/providers/nutanix/validator.go#L52

You can use this as ref https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/validator.go#L177-L184

Reply:
Fixed